app/modules/node.c

```diff
@@ -604,6 +604,22 @@ static int node_egc_setmode(lua_State* L) {
   legc_set_mode( L, mode, limit );
   return 0;
 }
+//
+// Lua: osprint(true/false)
+// Allows you to turn on the native Espressif SDK printing
+static int node_osprint( lua_State* L )
+{
+  if (lua_toboolean(L, 1)) {
+    system_set_os_print(1);
+    os_install_putc1((void *)uart0_write_char);
```

Comment:
This is bizarre, and maybe someone can explain it to me....

The driver/uart.c has

```
    // install uart1 putc callback
#ifndef NODE_DEBUG
    os_install_putc1((void *)uart1_write_char);
#endif
```

in it. I.e. it points the output to uart1 (and uart0 is the one connected to the USB). I don't know if this was done to really suppress all the SDK output, or why.

I'd be happy to remove this code, but I'm nervous that we might see output from it. The other alternative is just to make the no output function just drop characters.

Your call...